examples/kernels.rs

```diff
@@ -1,4 +1,5 @@
-use cudarc::{jit::*, prelude::*};
+#![feature(generic_const_exprs)]
```

Comment:
this is required to compile, even though no const generics are used directly ...

Reply:
Ahh I see just because the crate itself uses them in pub api?

Reply:
because the return type of one used function uses const generics. if you remove this line you'll get weird type mismatch errors